pyppeteer/page.py

```diff
@@ -85,113 +66,115 @@ class Page(EventEmitter):
     )
 
     @staticmethod
-    async def create(client: CDPSession, target: 'Target',
-                     ignoreHTTPSErrors: bool, defaultViewport: Optional[Dict],
-                     screenshotTaskQueue: list = None) -> 'Page':
+    async def create(
+            client: CDPSession,
+            target: 'Target',
+            ignoreHTTPSErrors: bool,
+            defaultViewport: Optional[Dict],
+            screenshotTaskQueue: list = None
+    ) -> 'Page':
         """Async function which makes new page object."""
-        await client.send('Page.enable'),
-        frameTree = (await client.send('Page.getFrameTree'))['frameTree']
-        page = Page(client, target, frameTree, ignoreHTTPSErrors,
-                    screenshotTaskQueue)
-
-        await asyncio.gather(
-            client.send('Target.setAutoAttach', {'autoAttach': True, 'waitForDebuggerOnStart': False}),  # noqa: E501
-            client.send('Page.setLifecycleEventsEnabled', {'enabled': True}),
-            client.send('Network.enable', {}),
-            client.send('Runtime.enable', {}),
-            client.send('Security.enable', {}),
-            client.send('Performance.enable', {}),
-            client.send('Log.enable', {}),
+        page = Page(
+            client=client,
+            target=target,
+            ignoreHTTPSErrors=ignoreHTTPSErrors,
+            screenshotTaskQueue=screenshotTaskQueue
         )
-        if ignoreHTTPSErrors:
-            await client.send('Security.setOverrideCertificateErrors',
-                              {'override': True})
+        await page._initialize()
         if defaultViewport:
             await page.setViewport(defaultViewport)
         return page
 
-    def __init__(self, client: CDPSession, target: 'Target',  # noqa: C901
-                 frameTree: Dict, ignoreHTTPSErrors: bool,
-                 screenshotTaskQueue: list = None) -> None:
+    def __init__(
+            self,
+            client: CDPSession,
+            target: 'Target',  # noqa: C901
```

Comment:
Let's not prematurely ignore function complexity requirements
```suggestion
            target: Target,
```